Author: <hesso@pool.math.tu-berlin.de>
Description: Add a missing include to get a few prototypes that caused build
issues on kfreebsd-amd64. (See #357232)
--- a/osdef.h.in
+++ b/osdef.h.in
@@ -106,6 +106,7 @@
 extern char *tgoto __P((char *, int, int));
 
 #ifdef POSIX
+#include <string.h>
 extern int   setsid __P((void));
 extern int   setpgid __P((int, int));
 extern int   tcsetpgrp __P((int, int));
